packit/schema.py

```diff
@@ -302,6 +323,20 @@ class PackageConfigSchema(MM23Schema):
     def ordered_preprocess(self, data, **kwargs):
         data = self.rename_deprecated_keys(data, **kwargs)
         data = self.specfile_path_pre(data, **kwargs)
+        data = self.add_defaults_for_jobs(data, **kwargs)
+        return data
+
+    @staticmethod
+    def add_defaults_for_jobs(data, **_):
+        """
+        add all the fields (except for jobs) to every job so we can process only jobconfig in p-s
+        """
+        for job in data.get("jobs", []):
```

Comment:
Just wondering if this works with the default jobs.

Reply:
good point

tests were failing on it, am still fixing them